modin/utils.py

```diff
@@ -258,30 +297,31 @@ def _replace_doc(
     target_doc = target_obj.__doc__ or ""
     overwrite = overwrite or not target_doc
     doc = source_doc if overwrite else target_doc
-    apilink = [apilink] if not isinstance(apilink, list) and apilink else apilink
+    apilink_l: Optional[List[str]]
+    apilink_l = [apilink] if not isinstance(apilink, list) and apilink else apilink  # type: ignore
 
     if parent_cls and not attr_name:
         if isinstance(target_obj, property):
-            attr_name = target_obj.fget.__name__
+            attr_name = target_obj.fget.__name__  # type: ignore
         elif isinstance(target_obj, (staticmethod, classmethod)):
             attr_name = target_obj.__func__.__name__
         else:
-            attr_name = target_obj.__name__
+            attr_name = target_obj.__name__  # type: ignore
```

Comment:
`modin/utils.py:309:25: error: "object" has no attribute "__name__"; maybe "__ne__" or "__new__"?  [attr-defined]`

This branch seems to assume that `target_obj` is `callable`, so maybe it would be better to add an assertion to that effect here.

Reply:
It seems like not all `callable`s have this field: https://stackoverflow.com/questions/33576932/do-all-callables-have-name
I'll leave this ignore in for now.